Dockerfile

```diff
@@ -3,6 +3,7 @@
 #     CGO_ENABLED=0 go build -a -tags netgo
 #     docker build --rm=true -t plugins/drone-cache .
 
-FROM gliderlabs/alpine:3.1
+FROM alpine:3.4
```

Comment:
And even that will change as it doesn't provide a shell or something for debugging